README.adoc

```diff
@@ -34,7 +34,7 @@ Stejně tak pro citace používejte http://biblatex-biber.sourceforge.net/[Biber
 === Věnujte minimálně den přípravě sazby (před samotným psaním)
 
 Setkávám se s tím, že studenti něco nějak napíšou a před odevzdáním se to snaží naprasit do školní latexové šablony.
-Doporučuji předtím, než začnete psát, tomu (Xe)LaTexu minimálně den věnovat a všechno si hezky připravit.
+Doporučuji předtím, než začnete psát, se tomu (Xe)LaTexu minimálně den věnovat a všechno si hezky připravit.
```

Comment:
věnovat komu čemu den.

Reply:
Ale chybí tam sloveso, v tomto pořadí se bez zvratného se neobejdeš...
A co: Doporučuji, před tím, než začnete psát, minimálně den věnovat (Xe)LaTexu a všechno si hezky připravit.

Reply:
Doporučuji LaTeXu věnovat minimálně den. Pořád mi to přijde OK.

Reply:
tak možná mi tam vadí to 'tomu'... ale OK :)

Reply:
c5efc4a

Reply:
Super!